var/spack/repos/builtin/packages/py-mypy/package.py

```diff
@@ -22,7 +22,7 @@ class PyMypy(PythonPackage):
 
     depends_on("python@3.5:", type=("build", "run"))
     depends_on('py-setuptools', type=('build', 'run'))
-    depends_on('py-typed-ast@1.4.0:1.4', type=('build', 'run'))
+    depends_on('py-typed-ast@1.4.0:2', type=('build', 'run'))
```

Comment:
According to the setup.py:
```python
      install_requires=["typed_ast >= 1.4.0, < 1.5.0; python_version<'3.8'",             
````
this is right, although it needs a small tweak:
```suggestion
    depends_on('py-typed-ast@1.4.0:1.4', when='@0.900: ^python@:3.7', type=('build', 'run'))
    depends_on('py-typed-ast@1.4.0:1.4', when='@:0.899', type=('build', 'run'))
```

Reply:
was looking only at the latest `setup.py`, my bad